src/pages/index.js

```diff
@@ -68,18 +175,26 @@ const profileAddPopup = new PopupWithForm(
   "#profile-add-modal",
   handleCardFormSubmit
 );
+const deleteCardPopup = new PopupDeleteCard("#delete-image-modal");
+const profileImagePopup = new PopupWithForm(
+  "#edit-profile-img-modal",
+  handleProfileImgFormSubmit
+);
 
 previewImagePopup.setEventListeners();
 profileEditPopup.setEventListeners();
 profileAddPopup.setEventListeners();
+deleteCardPopup.setEventListeners();
+profileImagePopup.setEventListeners();
 
 // form validation
-
 const editFormValidator = new FormValidator(config, editForm);
```

Comment:
**Might be better**


You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```